arangod/Aql/OptimizerRulesFeature.cpp

```diff
@@ -374,41 +375,59 @@ std::unordered_set<int> OptimizerRulesFeature::getDisabledRuleIds(
     }
     if (name[0] == '-') {
       // disable rule
```

Comment:
this comment should now be removed, as the function name is showing whats happening, below too.